rewrite-groovy/src/test/java/org/openrewrite/groovy/tree/MethodInvocationTest.java

```diff
@@ -262,4 +262,31 @@ static void main(String[] args) {
           )
         );
     }
+
+    @Issue("https://github.com/openrewrite/rewrite/issues/4703")
+    @Test
+    void insideParentheses() {
+        rewriteRun(
+          groovy(
+            """              
+              static def foo(Map map) {
+                  ((map.containsKey("foo"))
+                      && ((map.get("foo")).equals("bar")))
+              }
+              """
+          )
+        );
+    }
+
+    @Issue("https://github.com/openrewrite/rewrite/issues/4703")
```

Comment:
Don't think this example came from this issue?

Reply:
Nope, different issue but very same reason. So I marked the issue as duplicate and put given test in the PR.